receiver/azureeventhubreceiver/README.md

```diff
@@ -50,6 +50,18 @@ attribute names are copied without any changes.
 
 Default: `false` (semantic conventions are not applied)
 
+### time_format (optional)
+
+All supported time format for logs, metrics and traces. Default is empty string array, which means using the current iso8601 parser. The format is based on https://pkg.go.dev/time#Layout. If no time-zone info, will use UTC time.
```

Comment:
why multiple? Are you saying we will try them in order? If none work, do we default back to iso8601? If all fails, what is the behavior?

Reply:
Currently, the logic is either trying formats in order or using the default iso8601 if no formats set. I think your approach is better, changed to try them in order, and fallback to iso8601 if all fails. If the iso8601 also failed, it'll throw the `Unable to convert timestamp from log` as before.

Reply:
ok!

Reply:
the path of least surprise is a good idea. Thank you!